javascript-data-model/order-history.js

```diff
@@ -0,0 +1,46 @@
+var orderHistory = [
+  {
+    orderNumber: '114-3941689-8772232',
+    numberofItems: '1',
+    item: 'JavaScript for impatient programmers',
+    cost: '$31.55',
+    orderPlaced: 'August 4, 2020',
+    totalCost: '$34.00',
+    shippedTo: 'JS Masher',
+    deliveredOn: 'Aug 8, 2020',
+    returnWindow: 'Sept 7, 2020'
+  },
+  {
+    orderNumber: '113-9984268-1280257',
+    numberofItems: '1',
+    item: 'The Timeless Way of Building',
+    cost: '$41.33',
+    orderPlaced: 'July 19, 2020',
+    totalCost: '$44.53',
+    shippedTo: 'JS Masher',
+    deliveredOn: 'Jul 20, 2020',
+    returnWindow: 'Aug 19, 2020'
+  },
+  {
+    orderNumber: '114-2875557-9059409',
+    numberofItems: '1',
+    item: 'Gamecube Controller Adapter',
+    cost: '$15.98',
+    orderPlaced: 'July 4, 2020',
+    totalCost: '$17.22',
+    shippedTo: 'JS Masher',
+    deliveredOn: 'Jul 7, 2020',
+    returnWindow: 'Aug 5, 2020'
+  },
+  {
+    orderNumber: '113-2883177-2648248',
+    numberofItems: '2',
+    item: ['GameCube Controller', 'The Art of Sql'],
+    cost: ['$94.95', '$33.99'],
```

Comment:
Notice how this entry has arrays for values, but the rest of your entries have strings for values. This is not a consistent data model, which will cause issues if you ever tried to use this data. Make sure that your format matches for all orders.